.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,42 @@
+name: GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ./client
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+          cache-dependency-path: client/yarn.lock
+
+      - name: yarn ci and build
+        env:
+          CI: false
+          PUBLIC_URL: "/swayswap-demo"
```

Comment:
This is to set the URL to `https://fuellabs.github.io/swayswap-demo`? I think GitHub pages will automatically set it to that URL, so you might be doubling up here? Not sure.

Reply:
Correct, it will auto assign based on the name of the repository!

Reply:
This is for the react build, we need to inform the router to work with the base path ‘swayswap-demo’.